x-pack/plugins/security_solution/server/lib/detection_engine/signals/threat_mapping/enrich_signal_threat_matches.ts

```diff
@@ -134,7 +169,7 @@ export const enrichSignalThreatMatches = async (
     ? signalMatchesArg
     : uniqueHits.map((signalHit) => ({
         signalId: signalHit._id,
-        queries: extractNamedQueries(signalHit),
+        queries: extractNamedQueries(signalHit) as ThreatMatchNamedQuery[],
```

Comment:
Thanks for notice!

Removed whole ```uniqueHits.map((signalHit) => ({
        signalId: signalHit._id,
        queries: extractNamedQueries(signalHit),
        queries: extractNamedQueries(signalHit) as ThreatMatchNamedQuery[],``` sections, because it's not used anymore